src/features/game/lib/gameMachine.ts

```diff
@@ -219,6 +224,41 @@ export function startGame(authContext: Options) {
             },
           },
         },
+        loaded: {
+          always: [
+            {
+              target: "announcing",
+              cond: () => {
+                const lastRead = localStorage.getItem("announcementLastRead");
```

Comment:
Consider moving this into a separate data storage lib file.

We've got some repeated use of the `announcementLastRead` key which can also be abstracted into a constant to prevent future bugs.